frontend/src/features/projects/erp/forms/EnhancedReferralCompleteForm.tsx

```diff
@@ -48,6 +49,28 @@ const EnhancedReferralCompleteForm = ({
   return (
     <Container fluid className="EnhancedReferralCompleteForm">
       <h3>Enhanced Referral Process Complete</h3>
+      <Form.Row>
+        <Form.Label column md={4}>
+          Interest Received On
+        </Form.Label>
+        <FastDatePicker
+          outerClassName="col-md-2"
+          formikProps={formikProps}
+          disabled={isReadOnly}
+          field="interestedReceivedOn"
+        />
+      </Form.Row>
+      <Form.Row>
+        <Col>
+          <ProjectNotes
+            field="interestFromEnhancedReferralNote"
+            label="Interest Note"
+            disabled={isReadOnly}
+            outerClassName="col"
```

Comment:
is col just making this a 100% width field?

Reply:
without it the text area doesn't align with the other inputs on the form.